demo/vl-map-layer-switcher.html

```diff
@@ -0,0 +1,141 @@
+                        <vl-map-side-sheet>
+                            <vl-map-layer-switcher>
+                                <vl-checkbox data-vl-label="Kaartlaag" data-vl-layer="layer-1"></vl-checkbox>
+                            </vl-map-layer-switcher>
+                        </vl-map-side-sheet>
+                        <vl-map-overview-map></vl-map-overview-map>
+                        <vl-map-baselayer-grb-gray id="baselayer-grb-gray"></vl-map-baselayer-grb-gray>
+                        <vl-map-baselayer-grb id="baselayer-grb"></vl-map-baselayer-grb>
+                        <vl-map-baselayer-grb-ortho></vl-map-baselayer-grb-ortho>
+                        <vl-map-layer data-vl-name="layer-1" data-vl-features='
+                            {
+                                "type": "FeatureCollection",
+                                "features": [{
+                                    "type": "Feature",
+                                    "geometry": {
+                                        "type": "Point",
+                                        "coordinates": [149055.0,199908.0]
+                                    }
+                                }]
+                            }'>
+                            <vl-map-layer-circle-style data-vl-color="black"></vl-map-layer-circle-style>
+                        </vl-map-layer>
+                        <vl-map-layer data-vl-name="layer-2" data-vl-features='
+                            {
+                                "type": "FeatureCollection",
+                                "features": [{
+                                    "type": "Feature",
+                                    "geometry": {
+                                        "type": "Point",
+                                        "coordinates": [154055.0,199908.0]
+                                    }
+                                }]
+                            }'>
+                            <vl-map-layer-circle-style data-vl-color="yellow"></vl-map-layer-circle-style>
+                        </vl-map-layer>
+                        <vl-map-layer data-vl-name="layer-3" data-vl-features='
+                            {
+                                "type": "FeatureCollection",
+                                "features": [{
+                                    "type": "Feature",
+                                    "geometry": {
+                                        "type": "Point",
+                                        "coordinates": [159055.0,199908.0]
+                                    }
+                                }]
+                            }'>
+                            <vl-map-layer-circle-style data-vl-color="red"></vl-map-layer-circle-style>
+                        </vl-map-layer>
+                    </vl-map>
+                </div>
+            </div>
+        </vl-demo>
+    </vl-demo-page>
+</body>
+
+</html>
```

Comment:
Wat ik in de demo mis, maar dat is misschien een persoonlijke mening, is een voorbeeld van hoe je in de layerswitcher kaartlagen hebt die default niet zijn aangevinkt.

Reply:
Op dit moment is het nog niet mogelijk om een <vl-map-layer/> aan te maken die default hidden is. In een volgende PR is het wel mogelijk om te werken met resoluties, waardoor dit gedeeltelijk bereikt kan worden. Als dit een ontbrekende feature is, kan dit best toegevoegd worden aan de lijst van features.

Reply:
die features is al in bestelling: Als ontwikkelaar wil ik een laag kun (de)activeren